operations/reservations.md

```diff
@@ -239,6 +243,13 @@ Returns all reservations specified by any identifier, customer or other filter.
 * `Connector`
 * `Navigator`
 
+#### Reservation purpose
+
+* `Leisure`
+* `Business`
+* `Student`
+* ...
```

Comment:
Not sure about the potential expansion, it's not consistent across the documentation.

Reply:
@jirrick Why, that means if we will add new Purpose, we will not be able to serialize in the API

Reply:
We need to distinguish between several types

1) "Business-critical", Those emus have a direct impact on the whole flow. We cannot change them, update them Reservation State, Payment state,...
2) I will call them "Categorization", not sure if this is the correct word, but those are used to allow us to get some better classification of data. Those can be added any time. Like CustomerClassification, that is defined in the same way. Otherwise, we would not be able to ever add another classification for customers. The same for this Purpose.